src/core/jupyter/jupyter.ts

```diff
@@ -1930,9 +1930,15 @@ function mdImageOutput(
     ? (data as string[]).join("")
     : data as string;
 
-  // base64 decode if it's not svg
   const outputFile = join(options.assets.base_dir, imageFile);
-  if (mimeType !== kImageSvg) {
+  if (
+    // base64 decode if it's not svg
+    mimeType !== kImageSvg ||
+    // or if it is encoded svg; this could happen when used in embed context,
+    // as Pandoc will generate ipynb with base64 encoded svg data
+    // https://github.com/quarto-dev/quarto-cli/issues/9793
+    !/<svg/.test(imageText)
```

Comment:
@cscheid @gordonwoodhull I don't know if this is robust or not. It seems simple enough. 

Otherwise, I was thinking of testing for base64 string ? 
Either using regex 
````typescript
  function isBase64(value: string) {
    return /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})$/
      .test(value.replace(/\s+/gm, ""));
  }
````
Or a encode decode check 

````typescript
  function isBase64(value: string) {
    value = value.replace(/\s+/gm, "");
    try {
      return encodeBase64(decodeBase64(value)) == value;
    } catch {
      return false;
    }
  }
````

Would this utility be useful or too fragile for the regex one, and too computing intensive for the second one (I don't know if decode + encode can impact performance) ?

Reply:
Given that 
* the code already assumes that every image mime type except SVG is base64 encoded
* `<` is not a valid base64 chararacter

I think your fix is correct, and there is no need for the other stuff. I would probably pin it with `/^<svg/` but it doesn't matter.

Reply:
> I would probably pin it with /^<svg/ but it doesn't matter.

This is what I tried first 😅 https://github.com/quarto-dev/quarto-cli/pull/9795/commits/e607284655af9752be3ac09f999bba9d9f821710 but test failed, because it seems some figures output in ipynb will include a whole HTML doc, so not starting with `<svg` - I know - not expected 😞